src/ui/src/builder/modals/UnsavedChangesModal.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<WdsModal
+		title="Unsaved Changes"
+		description="You have unsaved changes. What would you like to do?"
+		:actions="modalActions"
+		display-close-button
+		@close="() => emits('cancel')"
```

Comment:
You don't need to wrap function in Vue

```suggestion
		@close="emits('cancel')"
```